proto/osmosis/gamm/pool-models/stableswap/stableswapPool.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+package osmosis.gamm.stableswap.v1beta1;
+
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+import "cosmos/auth/v1beta1/auth.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v7/x/gamm/pool-models/stableswap";
+
+// PoolParams defined the parameters that will be managed by the pool
+// governance in the future. This params are not managed by the chain
+// governance. Instead they will be managed by the token holders of the pool.
+// The pool's token holders are specified in future_pool_governor.
+message PoolParams {
+  string swapFee = 1 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.moretags) = "yaml:\"swap_fee\"",
+    (gogoproto.nullable) = false
+  ];
+  string exitFee = 2 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.moretags) = "yaml:\"exit_fee\"",
+    (gogoproto.nullable) = false
+  ];
+}
+
+message Pool {
```

Comment:
nit: proto doc :p

Reply:
Not really sure what doc to put here yet, I think It'll become clearer when I go to make one go-side after an initial impl of the math. Added a placeholder for now though